.travis.yml

```diff
@@ -3,20 +3,23 @@
 language: python
 
 python:
-  - 2.7
   - 3.5
+  - 3.6
 
 env:
-  - TOXENV=django18
-  - TOXENV=django19
-  - TOXENV=django110
   - TOXENV=django111
+  - TOXENV=django20
+  - TOXENV=django21
+  - TOXENV=django22
 
 matrix:
   include:
+    - python: 2.7
```

Comment:
You need to exlude for py2.7 with django20-21-22

Reply:
I've added it in `matrix` section